### DISCUSSION TITLE
Localized asset fields

### DISCUSSION THREAD

**Discussion by simonhamp** [statamic/cms#4904] [Troubleshooting] [answered]
Something of a continuation/re-opening of [this Forum thread](https://statamic.com/forum/5263-localizaton-for-asset-fields)

Is there an appetite for making asset fields actually localisable? It seems that they are via the Blueprint (you can click the little globe icon), but in practice they're not - when editing via the Asset Browser, it seems to edit only one piece about the asset and this is always shared regardless of selected site/locale.

Maybe they _are_ supposed to be and this is just a bug that needs fixing?

**Comment by simonhamp** [comment, marked as answer]
Ah, I've just been through issues and statamic/ideas and come across this: https://github.com/statamic/ideas/issues/223

So if you would like to see this, please give that an upvote.

**Comment by zehfred** [comment]
It's definitely something that needs to happen. I thought it was a bug too. What's the point of localizable: true if it doesn't work? 🤔

**Reply by jasonvarga** [reply]
Fair point - we should remove the globe when editing blueprints that aren't localizable like assets, forms, users, etc.

Right now you just get the globe everywhere because it's the same component used in each section.
